### ISSUE TITLE
Guided tours

### ISSUE THREAD

**Issue by Jazzpirate** [KWARC/FoMID#16]
Easy to implement in theory: Take the topologically sorted list of dependencies of a concept/document and present to reader.

Desirable: Have a user permanently mark modules as *trivial*/*known*, so that they are omitted from guided tours for *that specific user*. Easy to do in an IDE, on MathHub that would require accounts, I guess?

**Comment by kohlhase** [comment]
The more I think about them, the more I dislike the name "guided tours", Even though I coined it in 1999. 

There is also a dual concept to guided tours: "excursions", which -- given a dependency graph identify "interesting" items that are close to and well-connected with marked-as-known material. They are "guided tours" with a free goal.

**Comment by kohlhase** [comment]
> Easy to do in an IDE, on MathHub that would require accounts, I guess?

indeed, and there are many many more ways of "user modeling" to predict what the user actually knows than only "manually marking".

**Comment by Jazzpirate** [comment]
> indeed, and there are many many more ways of "user modeling" to predict what the user actually knows than only "manually marking".

Do you have concrete ideas in that direction already?

**Comment by kohlhase** [comment]
> Do you have concrete ideas in that direction already?

there were many ideas over the years, but no specific systems (apart from a partial implemen tation named PanthaRhei by Christine Müller.
